cassandra/Dockerfile

```diff
@@ -39,24 +38,28 @@ ENV CASSANDRA_VERSION="2.2.1" \
 # Become the root user to be able to install and setup Cassandra under /opt
 USER root
 
+#
+RUN yum install -y -q bind-utils && \
+    yum clean all
+
+COPY apache-cassandra-2.2.4-bin.tar.gz.md5 \
```

Comment:
Are we only able to get this md5 by checking it in locally? If there was a way to parameterize, we could propagate the CASSANDRA_VERSION throughout the Dockerfile